DDCore/include/XML/UnicodeValues.h

```diff
@@ -14,6 +14,8 @@
 //
 //==========================================================================
 
+#ifdef UNICODE
```

Comment:
clang complains that UNICODE is not defined, so this header file only makes sense if the one that defines UNICODE is included first